zk-collectd.py

```diff
@@ -101,6 +101,8 @@ def _get_mntr_stats(self):
                 key, value = self._parse_line(line)
                 if key == "zk_server_state":
                     result["zk_is_leader"] = int(value != "follower")
+                elif key == "zk_peer_state":
+                    result["zk_peer_state"] = int(value.startswith("lead"))
```

Comment:
I'm not a zookeeper expert, but I think this field is to relay zap quorum state and there are multiple possible values so 0 or 1 are probably not what we're looking for w/ `zk_peer_state` as the metric name.

Reply:
https://github.com/apache/zookeeper/commit/cc900a3b05bc31a237753680c8b00dc5866df4b2

Reply:
This bug is less about adding support for this new metric and more about getting rid of the error.
You ok if we just skip it for now so it won't be sent to the SA? and then we can add this metric and enhancement?

Reply:
Sure, (temporarily) ignoring unsupported keys seems ok for now.